modules/rosa-sts-setting-up-environment.adoc

```diff
@@ -0,0 +1,252 @@
+[id="rosa-sts-setting-up-environment_{context}"]
+
+= Setting up the environment for STS
+
+Complete the following steps to set up your environment before creating your cluster using AWS security token Service (STS).
+
+.Prerequisites
+
+* Review and complete the deployment prerequisites and policies.
+* Create a link:https://cloud.redhat.com[Red Hat account], if you do not already have one. Then, check your email for a verification link. You will need these credentials to install ROSA.
+
+.Procedure
+
+. Log in to the Amazon Web Services (AWS) account that you want to use.
++
+It is recommended to use a dedicated AWS account to run production clusters. If you are using AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
++
+If you are using AWS Organizations and you need to have a Service Control Policy (SCP) applied to the AWS account you plan to use, these policies must not be more restrictive than the roles and policies required by the cluster.
++
+As part of the cluster creation process, `rosa` creates an `osdCcsAdmin` IAM user. To configure the AWS CLI, 'osdCcsAdmin' uses the IAM credentials you provide.
```

Comment:
```suggestion
```

Reply:
Clarification needed: Does the Note that follows this line  go away too? 
"This user has `Programmatic` access enabled and the `AdministratorAccess` policy attached to it.'